File: TSPractice/P19.ts
// Program to Print Pyramid Number Pattern

export let n = 5;
for (let i = 1; i <= n; i++) {
    let row = "";
    for (let j = 1; j <= n-i; j++) {
        row += "  ";
    }
    for (let k = 0; k < i; k++) {
        row += (i + k) + " ";
    }
    for (let k = i - 2; k >= 0; k--) {
        row += (i + k) + " ";
    }
    console.log(row.trimEnd());
}

File: Java/src/P19.java

// Program to Print Pyramid Number Pattern

public class P19 {
    public static void main(String[] args) {
        // print decresing spaces
        // print number
        int n = 5;
        for (int i = 1; i <= n; i++) {
            // Print spaces
            for (int j = 1; j <= n - i; j++) {
                System.out.print(" ");
            }

            // Print numbers
            for (int k = 1; k <= i; k++) {
                System.out.print(k + " ");
            }

            // Move to the next line
            System.out.println();
        }
    }
}
